teachopencadd/talktorials/T003_compound_unwanted_substructures/talktorial.ipynb

```diff
@@ -122,23 +122,17 @@
    "cell_type": "code",
```

Comment:
<p>Could you please run <code>black-nb</code> over this notebook (should streamline the formatting with the rest of the notebooks):</p><p> </p><pre class="ql-syntax" spellcheck="false">black-nb -l 99 teachopencadd/talktorials/T003_compound_unwanted_substructures/talktorial.ipynb
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>